authors/apps/utils/messages.py

```diff
@@ -5,3 +5,9 @@
     'permission_denied': "You do not have permission to perform this action",
     'delete_msg': "{} successfully deleted"
 }
+
+
+favorite_actions_messages = {
+    'favorited':'Article has been added to favorites',
+    'un_favorited': 'Article has been removed from favorites'
+}
```

Comment:
These can be joined into a single message with help of the `.format()` `string` method

Reply:
But I guess it can be done during the code cleanup session

Reply:
I get your point but in this case, I would want to render a different message based on the action performed. Let me see if I can refactor to meet your request.